src/components/Nav/NewCollectiveModal.vue

```diff
@@ -231,13 +235,17 @@ export default {
 
 	watch: {
 		newCollectiveName(val) {
-			if (!val || this.newCollectiveName.length > 2) {
+			if (!val || (this.newCollectiveName.length > 2 && this.newCollectiveName.length < 128)) {
```

Comment:
I think the second check need to go into a separate `else if` block. Imagine that I had a name with more than 127 chars, so `nameIsToLong` is `true`. Then I remove all characters but one in a single step (by selecting all other characters and hitting <kbd>Del</kbd>). Now `newCollectiveName` is changed, so the watcher triggers. Now this combined condition is false, since the collective name has not more than two characters. So you'll end up with both `nameIsTooShort` and `nameIsTooLong` being `true`.